cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -112,21 +152,20 @@ func (p *nfsProvisioner) Delete(volume *v1.PersistentVolume) error {
 	// Determine if the "archiveOnDelete" parameter exists.
 	// If it exists and has a false value, delete the directory.
 	// Otherwise, archive it.
-	archiveOnDelete, exists := storageClass.Parameters["archiveOnDelete"]
-	if exists {
-		archiveBool, err := strconv.ParseBool(archiveOnDelete)
-		if err != nil {
-			return err
-		}
-		if !archiveBool {
-			return os.RemoveAll(oldPath)
-		}
-	}
+	onDelete := storageClass.Parameters["onDelete"]
```

Comment:
this would be backward incompatible

Reply:
@jackielii , not really, the default move solution is left. We wanted to propose adding a new parameter for similar operations.
However, if this is indeed a problem, the solution is to save the two parameters for full integration.